### ISSUE TITLE
Epithelial hyperplasia on biopsy

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10072]
**Preferred term label:**
Epithelial hyperplasia on biopsy

**Synonyms**

**Definition (free text, please give PubMed ID)**
https://www.ncbi.nlm.nih.gov/medgen/C0014599

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
 Intraductal papilloma

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
same comment as https://github.com/obophenotype/human-phenotype-ontology/issues/10071

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding 
HP:0020328
Label:  Mammary ductal epithelial cell hyperplasia
Def: Abnormal (increased) proliferation of epithelial cells in a mammary duct. Intraductal papilloma is a benign tumor found within breast ducts. The abnormal proliferation of ductal epithelial cells causes growth. A solitary intraductal papilloma is usually found centrally posterior to the nipple, affecting the central duct.
